ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -957,3 +957,15 @@ def Svd(*, input, full_matrices=False, compute_uv=True, name=None):
         "tensorflow",
     )(map_raw_ops_alias(tf_frontend.general_functions.einsum))
 )
+
+
+@to_ivy_arrays_and_back
+@with_supported_dtypes(
+    {"2.13.0 and below": ("float16", "bfloat16", "float32", "float64")}, "tensorflow"
+)
+def Selu(*, features, name="Selu"):
+    alpha = 1.6732632423543772848170429916717
+    scale = 1.0507009873554804934193349852946
+    return scale * (
```

Comment:
we have a `ivy.selu` function, is there any reason we're doing this compositionally? thanks!